lib/logstash/plugin_mixins/aws_config.rb

```diff
@@ -21,6 +21,8 @@ def setup_aws_config
                                   "eu-west-1", "ap-southeast-1", "ap-southeast-2",
                                   "ap-northeast-1", "sa-east-1", "us-gov-west-1"], :default => US_EAST_1
 
+    config :s3_host, :validate => :string, :default => 'amazonaws.com'
```

Comment:
I understand that `Ceph` only support the S3 API, but I think we should rename this option to `aws_host` to actually reflect that you changing the AWS host for your api endpoints. Can you also add a comment before the option to help the user understand what this setting actually does?